{
  "comments": [
    "I understand that the JIRA exception handling is better for when the stored credentials for whatever reason \"expire\", but when the JIRA website is unreachable / goes down new error crop up.  An error dialog displays the following message:\n\nAn Internal error occurred during: \"Get Changed Tasks\"\n\nHere is the stack trace:\n\norg.tigris.jira.core.service.exceptions.ServiceUnavailableException: Unable to connect to server.  Please check the URL.\n\tat org.tigris.jira.internal.core.service.soap.SoapJiraService$StandardLoginToken.getCurrentValue(SoapJiraService.java:575)\n\tat org.tigris.jira.internal.core.service.soap.SoapJiraService.getServerInfo(SoapJiraService.java:159)\n\tat org.tigris.jira.core.service.CachedRpcJiraServer.initializeServerInfo(CachedRpcJiraServer.java:409)\n\tat org.tigris.jira.core.service.CachedRpcJiraServer.getServerInfo(CachedRpcJiraServer.java:458)\n\tat org.eclipse.mylar.internal.jira.core.JiraOfflineTaskHandler.getChangedSinceLastSync(JiraOfflineTaskHandler.java:303)\n\tat org.eclipse.mylar.tasks.ui.RepositorySynchronizationManager$GetChangedTasksJob.run(RepositorySynchronizationManager.java:191)\n\tat org.eclipse.core.internal.jobs.Worker.run(Worker.java:58)\n\nIt wouldn\u0027t be so bad, except that I get this error dialog during every synchronization interval.\n\nI guess this precipitates - or rather, adds more emphasis to - the need to be able to put certain repositories into offline mode.",
    "And may I say that stack trace looks absolutely disgusting with the forced line breaks in the mylar editor.",
    "Created an attachment (id\u003d54762)\nGet Changed Tasks\n\n",
    "Created an attachment (id\u003d54823)\nhandle runtime exceptions from jira core services\n\nThese shouldn\u0027t be runtime exceptions in first place.\n\nPS: It is weird, that I can attach files (images), but attachment submission does fail every time when I am trying to submit a patch.\n",
    "Created an attachment (id\u003d54824)\ntest\n\n",
    "Created an attachment (id\u003d54826)\ntest\n\n",
    "Sorry about these tests. Apparently attachment submission is only failing when submitting a patch for the clipboard.",
    "Created an attachment (id\u003d54862)\nupdated patch to resolve conflicting changes\n\n",
    "Patch applied.",
    "Resolve?",
    "I can\u0027t test this.  My JIRA site went down as a fluke, and I don\u0027t have administrative access of it.  I thought of disabling my network adapter to test, but those are likely two completely different error paths.",
    "That is how I tested it. It fail with the same stack trace as you shown. There is other errors in the error log, but those require changes in jira sercices and can\u0027t be done now.",
    "Closing it as fixed. Joseph, if you still can reproduce this please reopen this issue."
  ],
  "commentCreationDates": [
    "2006-11-30T05:01:43+01:00",
    "2006-11-30T05:21:10+01:00",
    "2006-11-30T05:21:37+01:00",
    "2006-11-30T20:19:51+01:00",
    "2006-11-30T20:20:28+01:00",
    "2006-11-30T20:23:41+01:00",
    "2006-11-30T20:24:54+01:00",
    "2006-12-01T04:02:14+01:00",
    "2006-12-01T18:45:23+01:00",
    "2006-12-01T19:09:15+01:00",
    "2006-12-01T19:13:26+01:00",
    "2006-12-01T19:19:52+01:00",
    "2007-01-01T22:30:28+01:00"
  ],
  "traces": [
    {
      "exceptionType": "org.tigris.jira.core.service.exceptions.ServiceUnavailableException",
      "message": "Unable to connect to server.  Please check the URL.",
      "elements": [
        {
          "method": "org.tigris.jira.internal.core.service.soap.SoapJiraService$StandardLoginToken.getCurrentValue",
          "source": "SoapJiraService.java:575"
        },
        {
          "method": "org.tigris.jira.internal.core.service.soap.SoapJiraService.getServerInfo",
          "source": "SoapJiraService.java:159"
        },
        {
          "method": "org.tigris.jira.core.service.CachedRpcJiraServer.initializeServerInfo",
          "source": "CachedRpcJiraServer.java:409"
        },
        {
          "method": "org.tigris.jira.core.service.CachedRpcJiraServer.getServerInfo",
          "source": "CachedRpcJiraServer.java:458"
        },
        {
          "method": "org.eclipse.mylar.internal.jira.core.JiraOfflineTaskHandler.getChangedSinceLastSync",
          "source": "JiraOfflineTaskHandler.java:303"
        },
        {
          "method": "org.eclipse.mylar.tasks.ui.RepositorySynchronizationManager$GetChangedTasksJob.run",
          "source": "RepositorySynchronizationManager.java:191"
        },
        {
          "method": "org.eclipse.core.internal.jobs.Worker.run",
          "source": "Worker.java:58"
        }
      ],
      "number": 0,
      "commentIndex": 0,
      "bugId": "166309",
      "date": "2006-11-30T05:01:43+01:00",
      "product": "Mylyn",
      "component": "Bugzilla",
      "severity": "normal"
    }
  ],
  "groupId": "166309",
  "bugId": "166309",
  "date": "2006-11-30T05:01:43+01:00",
  "product": "Mylyn",
  "component": "Bugzilla",
  "severity": "normal"
}